tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -1,3 +1,5 @@
+using FakeItEasy.Tests.TestHelpers;

```

Comment:
Ah, shoot. I approved before noticing. If I can bother you for one more change, @skarum, could this using go down with its friends, after `FakeItEasy.Creation`?